libs/sql-parser/src/test/java/io/crate/sql/parser/TestSqlParser.java

```diff
@@ -185,7 +186,7 @@ public void testEmptyExpression() {
         assertThatThrownBy(
             () -> SqlParser.createExpression(""))
             .isExactlyInstanceOf(ParsingException.class)
-            .hasMessageStartingWith("line 1:1: mismatched input '<EOF>' expecting {'(', '[', '[]', '{',");
+            .hasMessageStartingWith("line 1:1: mismatched input '<EOF>' expecting {");
```

Comment:
Why has this changed regarding the `expecting` part?

Reply:
it is changed due to the commits and also `mismatched input...` seemed what was important here.